src/Bundler/PipelineBundler.php

```diff
@@ -60,16 +60,24 @@ public function execute(ReaderInterface $reader, WriterInterface $writer): void
             $output_require_file = new File($output_folder . '/require.js');
 
             if ($this->checkIfAnyChanged($output_require_file, [new Dependency($require_file)])) {
-                $this->logger->info('Writing require.js file to {name}', ['name' => $output_require_file->path]);
-
                 // Create an item for the file to write to disk.
                 $item = new ContentItem(
                     $require_file,
                     $output_require_file->getName(),
                     new StringReader($reader->read($require_file))
                 );
 
-                $writer->write($output_require_file, $this->runner->execute(RunnerType::UGLIFY, $item));
+                $content = $this->runner->execute(RunnerType::UGLIFY, $item);
+
+                $reporter->reportFileState($output_require_file, ReporterInterface::STATE_BUILT);
+                $reporter->reportFileSize($output_require_file, \strlen($content));
```

Comment:
Shouldn't we do a `reportFileContent` instead to avoid the `strlen` operation for a `NullReporter`?